CMakeLists.txt

```diff
@@ -13,3 +13,9 @@ set(USE_SYSTEM_CIMG ON CACHE BOOL "Set to OFF to use CImg library bundled in lib
 add_subdirectory(lib)
 
 add_subdirectory(src)
+
+option(PACKAGE_TESTS "Build the tests" ON)
```

Comment:
I don't understand why it's called `PACKAGE_TESTS`, it indicates that this would package the tests, e.g., using CPack.

Instead of inventing our own variable for this, CMake's `BUILD_TESTING` should be used.

```
include(CTest)

if(BUILD_TESTING)
    add_subdirectory(tests)
endif()